index.html

```diff
@@ -14,6 +14,9 @@
   <li>
     {{ conference.date_start | date: "%Y-%m-%d" }}&nbsp;
     <a class="post-link" href="{{ conference.website }}">{{ conference.name }}</a>
+    {% if conference.location %}
+    &nbsp;<small><a href="http://maps.google.com/?q={{ conference.location }}">{{ conference.location }}</a></small>
```

Comment:
I'm not sure the link is actually useful here. The location is extremely granular and the visual distinction between this link and the link to the conference website is little which will likely lead to people clicking the wrong one.